addon/components/materialize-collection-item-secondary.js

```diff
@@ -0,0 +1,10 @@
+import Ember from 'ember';
+import layout from '../templates/components/materialize-collection-item-secondary';
+
+export default Ember.Component.extend({
```

Comment:
should probably make use of a base class. Everything except

```js
classNames: ['secondary-content'],
```

is already defined in 
````
addon/components/materialize-collection-item-link.js
````